ush/generate_FV3LAM_wflow.py

```diff
@@ -1057,7 +1053,7 @@ def generate_FV3LAM_wflow():
         NOMADS_script = os.path.join(USHdir, "NOMADS_get_extrn_mdl_files.h")
         run_command(
             f"""{NOMADS_script} {date_to_str(DATE_FIRST_CYCL,format="%Y%m%d")} \
-                      {CYCL_HRS} {NOMADS_file_type} {FCST_LEN_HRS} {LBC_SPEC_INTVL_HRS}"""
+                      {date_to_str(DATE_FIRST_CYCL,format="%H")} {NOMADS_file_type} {FCST_LEN_HRS} {LBC_SPEC_INTVL_HRS}"""
```

Comment:
Not sure how/if this worked before in the python version.

Reply:
We (code developers) should talk about NOMADS support in general within the App.

Reply:
Also, I always thought it should have been a task instead of something done during workflow generation.

Reply:
This definitely does *not* work in the current app (see line 1053 where the wrong filename is called), though I do think it should be fixed rather than deprecated. The reason it has not been tested (and therefore this bug was introduced) is due to the test limitations mentioned in #357.

@danielabdi-noaa Ideally that would be the case, but because it requires internet access this will not work on many machines' compute nodes. So if it is moved to the main workflow it will have to be designed carefully.

Reply:
Okay. Welp, that's def outside the scope of this PR, and it's a known issue.